tests/Bridge/Symfony/Validator/Metadata/Property/Restriction/PropertySchemaRegexRestrictionTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Tests\Bridge\Symfony\Validator\Metadata\Property\Restriction;
+
+use ApiPlatform\Core\Bridge\Symfony\Validator\Metadata\Property\Restriction\PropertySchemaRegexRestriction;
+use ApiPlatform\Core\Metadata\Property\PropertyMetadata;
+use ApiPlatform\Core\Tests\ProphecyTrait;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Constraints\Positive;
+use Symfony\Component\Validator\Constraints\Regex;
+
+/**
+ * @author Tomas Norkūnas <norkunas.tom@gmail.com>
```

Comment:
Really? :D

Reply:
I guess it's to honor your work! :stuck_out_tongue:

Reply:
Right, that was just a copy-paste-rename-tweak of PropertySchemaCountRestrictionTest, so I didn't feel like putting my name/email here 🙂